### ISSUE TITLE
The `set-output` command is deprecated and will be disabled soon. 

### ISSUE THREAD

**Issue by anuteresa** [voxmedia/github-action-slack-notify-build#71]

**Comment by lfg-will** [comment]
+1.  This is causing the following annotation to show for any jobs using this action:

> The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

**Comment by summitt** [comment]
+1 . Would be great if there could be a fix for this soon.

**Comment by timam** [comment]
+1

**Comment by vitalykarasik** [comment]
I created https://github.com/voxmedia/github-action-slack-notify-build/pull/82

**Comment by kevin-riquena** [comment]
+1

**Comment by jefflaporte** [comment]
@ebinmore Would you be able to merge #82 from @vitalykarasik ?

**Comment by CSHawley** [comment]
Just a reminder that [Slack Send ](https://github.com/marketplace/actions/slack-send) exists as an alternative. It looks to be developed in-house by the Slack team and they seem to be proactive about addressing breaking changes.

**Comment by vitalykarasik** [comment]
> Just a reminder that [Slack Send ](https://github.com/marketplace/actions/slack-send) exists as an alternative. It looks to be developed in-house by the Slack team.

You're right, but I prefer this plugin because I don't need to generate the whole notification's body, so I need less code in my workflow. (BTW, during migration from Jenkins to GHA I compared a few Slack plugins and couldn't find 'my ideal plugin'.  For example, in this one, I lack the ability to add a custom body. A few others don't support modern Slack integration. )

**Comment by aldenquimby** [comment]
This repo appears abandoned: https://github.com/voxmedia/github-action-slack-notify-build/pull/69#issuecomment-1370041457

@klamas1 would you be willing to merge and ship #82 and take over as the primary fork for this repo moving forward?

**Comment by klamas1** [comment]
> @klamas1 would you be willing to merge and ship #82 and take over as the primary fork for this repo moving forward?

Unfortunately, I rarely use github actions, so I won't be able to keep the fork up to date and timely update the versions

**Comment by justinr636** [comment]
@aldenquimby I can't commit to any further maintenance, but I have taken the changes from @vitalykarasik in PR #82, forked the code, and published it to a new marketplace action found here: https://github.com/marketplace/actions/gha-slack-notify-build

**Comment by rmagri** [comment]
+1
